plugins/modules/mysql_user.py

```diff
@@ -592,11 +633,15 @@ def user_mod(cursor, user, host, host_all, password, encrypted,
                 if module.check_mode:
                     return (True, msg)
                 if old_user_mgmt:
-                    cursor.execute("SET PASSWORD FOR %s@%s = %s", (user, host, encrypted_password))
+                    query = "SET PASSWORD FOR %s@%s = %s"
```

Comment:
I did a bit of research and enabling integration tests for MySQL 5.6 is not going to be trivial.  `old_user_mgmt` handles cases where MariaDB < 10.2 and MySQL < 5.7.  The only version in that set that is still supported is MySQL 5.6 (for another two weeks).  I don't see it being worthwhile spending any time on this.  As per the discussion is #91 it might be worthwhile to consider dropping this support and removing the code in the near future.

Unless there are other suggestions, I can go ahead and remove this change.  The downside is that until we remove MariaDB < 10.2 and MySQL < 5.7 code paths, there will be queries that can modify the database state, but will not be logged via the module.